interfaces/console/src/components/NodeOverviewTab/index.tsx

```diff
@@ -78,13 +86,16 @@ const NodeOverviewTab = ({
             handleAction={handleOnSelected}
           >
             <NodeStatItem
-              value={`${capitalize(selectedNode?.type.toLowerCase() ?? 'HOME')} Node`}
+              id={null}
```

Comment:
Why are we passing null here ?

Reply:
It part of behaviour. If `id` is not passed some of the content will not appear. I can convert it to `undefined` rather null